### ISSUE TITLE
Book covers display incorrectly when using a screen reader

### ISSUE THREAD

**Issue by Flameborn** [bookwyrm-social/bookwyrm#1038]
When using a screen reader, such as VoiceOver under MAcOS, book covers display as the book ID, rather than an image with the book title, publication date, etc.

Pages where the issue appears include the feed, or an author page.

Tested via VoiceOver under Safari, via b3e369cdba823413b81fec0b4f506a1909520a09

**Comment by mouse-reeve** [comment]
This is caused by usia aria-hidden on the covers - since they're usually links, the alt text was providing the link text, and it fell back on the book id when aria-hidden was set to true. For now, I'm removing aria-hidden from the covers, and which should be shown can be re-evaluated in the future

**Comment by mouse-reeve** [comment]
Thank you so much for flagging all these issues! This should be working again on bookwyrm.social

**Comment by Flameborn** [comment]
Thank you very much for the quick fix. This indeed works now.

I'll create new issues in case I find sections where the image should be hidden.
